certified-connectors/VitaCloud Quotes/apiDefinition.swagger.json

```diff
@@ -0,0 +1,575 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "VitaCloud Quotes",
+    "description": "A daily dose of inspirational quotes delivered to you application or website. Choose from a variety of themes and authors.",
+    "termsOfService": "https://www.vitacloud.co.uk",
+    "contact": {
+      "name": "VitaCloud Support",
+      "url": "https://www.vitacloud.co.uk",
+      "email": "support@vitacloud.co.uk"
+    },
+    "version": "v1"
+  },
+  "host": "apis.vitacloud.co.uk",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://www.vitacloud.co.uk"
+    },
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://www.vitacloud.co.uk/PrivacyPolicy"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "Lifestyle and Entertainment;Productivity"
+    }
+  ],
+  "paths": {
+    "/Authors": {
+      "get": {
+        "tags": [
+          "Authors"
+        ],
+        "summary": "Get all authors",
+        "description": "Sample request:    GET /Authors",
```

Comment:
@kapilmehtagit please remove the extra white spaces from the description. Please fix all the descriptions in the swagger file.